Description: Remove external references to training advertising 
 Training advertise contains links to external resources that may not be
 available or may be used for tracking users activity without their knowledge.
Author: Janos Guljas <janos@debian.org>
Last-Update: 2013-04-06

--- a/docsite/rst/index.rst
+++ b/docsite/rst/index.rst
@@ -3,16 +3,8 @@
 Ansible Documentation
 `````````````````````
 
-.. image:: http://ansible.cc/docs/_static/ansible_fest_2013.png
-   :alt: ansiblefest 2013
-   :target: http://ansibleworks.com/fest
 
-
-This page contains documentation about how to use `Ansible <http://ansible.cc>`_.  You may also be interested in taking a class:
-
-.. image:: http://www.ansibleworks.com.s3-website-us-east-1.amazonaws.com/img/banners/training.png
-   :alt: ansibleworks training
-   :target: http://www.ansibleworks.com/training/
+This page contains documentation about how to use `Ansible <http://ansible.cc>`_.  You may also be interested in taking a class http://www.ansibleworks.com/training/.
 
 Before we dive into playbooks, configuration management, deployment, and orchestration, we'll learn how to get Ansible installed and some
 basic information.  We'll go over how to execute ad-hoc commands in parallel across your nodes using /usr/bin/ansible.  We'll also see 
--- a/docsite/rst/api.rst
+++ b/docsite/rst/api.rst
@@ -1,10 +1,6 @@
 API & Integrations
 ==================
 
-.. image:: http://ansible.cc/docs/_static/ansible_fest_2013.png
-   :alt: ansiblefest 2013
-   :target: http://ansibleworks.com/fest
-
 
 There are several interesting ways to use Ansible from an API perspective.   You can use
 the Ansible python API to control nodes, you can extend Ansible to respond to various python events,
--- a/docsite/rst/bestpractices.rst
+++ b/docsite/rst/bestpractices.rst
@@ -1,10 +1,6 @@
 Best Practices
 ==============
 
-.. image:: http://ansible.cc/docs/_static/ansible_fest_2013.png
-   :alt: ansiblefest 2013
-   :target: http://ansibleworks.com/fest/
-
 
 Here are some tips for making the most of Ansible.
 
--- a/docsite/rst/contrib.rst
+++ b/docsite/rst/contrib.rst
@@ -1,10 +1,6 @@
 Ansible Resources
 =================
 
-.. image:: http://ansible.cc/docs/_static/ansible_fest_2013.png
-   :alt: ansiblefest 2013
-   :target: http://ansibleworks.com/fest
-
 
 User contributed playbooks, modules, and articles. This is a small
 curated list, but growing. Everyone is encouraged to add to this
--- a/docsite/rst/examples.rst
+++ b/docsite/rst/examples.rst
@@ -1,10 +1,6 @@
 Command Line Examples And Next Steps
 ====================================
 
-.. image:: http://ansible.cc/docs/_static/ansible_fest_2013.png
-   :alt: ansiblefest 2013
-   :target: http://ansibleworks.com/fest
-
 .. highlight:: bash
 
 The following examples show how to use `/usr/bin/ansible` for running
--- a/docsite/rst/gettingstarted.rst
+++ b/docsite/rst/gettingstarted.rst
@@ -1,10 +1,6 @@
 Getting Started
 ===============
 
-.. image:: http://ansible.cc/docs/_static/ansible_fest_2013.png
-   :alt: ansiblefest 2013
-   :target: http://ansibleworks.com/fest
-
 
 .. contents::
    :depth: 2
--- a/docsite/rst/glossary.rst
+++ b/docsite/rst/glossary.rst
@@ -1,10 +1,6 @@
 Glossary
 ========
 
-.. image:: http://ansible.cc/docs/_static/ansible_fest_2013.png
-   :alt: ansiblefest 2013
-   :target: http://ansibleworks.com/fest
-
 
 The following is a list (and re-explanation) of term definitions used elsewhere in the Ansible documentation.
 
--- a/docsite/rst/moduledev.rst
+++ b/docsite/rst/moduledev.rst
@@ -1,10 +1,6 @@
 Module Development
 ==================
 
-.. image:: http://ansible.cc/docs/_static/ansible_fest_2013.png
-   :alt: ansiblefest 2013
-   :target: http://ansibleworks.com/fest
-
 
 Ansible modules are reusable units of magic that can be used by the Ansible API,
 or by the `ansible` or `ansible-playbook` programs.
--- a/docsite/rst/modules.rst
+++ b/docsite/rst/modules.rst
@@ -1,10 +1,6 @@
 Ansible Modules
 ===============
 
-.. image:: http://ansible.cc/docs/_static/ansible_fest_2013.png
-   :alt: ansiblefest 2013
-   :target: http://ansibleworks.com/fest
-
 .. contents::
    :depth: 2
    :backlinks: top
--- a/docsite/rst/patterns.rst
+++ b/docsite/rst/patterns.rst
@@ -3,10 +3,6 @@
 Inventory & Patterns
 ====================
 
-.. image:: http://ansible.cc/docs/_static/ansible_fest_2013.png
-   :alt: ansiblefest 2013
-   :target: http://ansibleworks.com/fest
-
 
 Ansible works against multiple systems in your infrastructure at the
 same time.  It does this by selecting portions of systems listed in
--- a/docsite/rst/playbooks2.rst
+++ b/docsite/rst/playbooks2.rst
@@ -1,10 +1,6 @@
 Advanced Playbooks
 ==================
 
-.. image:: http://ansible.cc/docs/_static/ansible_fest_2013.png
-   :alt: ansiblefest 2013
-   :target: http://ansibleworks.com/fest
-
 
 Here are some advanced features of the playbooks language.  Using all of these features
 are not neccessary, but many of them will prove useful.  If a feature doesn't seem immediately
--- a/docsite/rst/playbooks.rst
+++ b/docsite/rst/playbooks.rst
@@ -1,10 +1,6 @@
 Playbooks
 =========
 
-.. image:: http://ansible.cc/docs/_static/ansible_fest_2013.png
-   :alt: ansiblefest 2013
-   :target: http://ansibleworks.com/fest
-
 
 .. contents::
    :depth: 2
--- a/docsite/rst/YAMLSyntax.rst
+++ b/docsite/rst/YAMLSyntax.rst
@@ -1,10 +1,6 @@
 YAML Syntax
 ===========
 
-.. image:: http://ansible.cc/docs/_static/ansible_fest_2013.png
-   :alt: ansiblefest 2013
-   :target: http://ansibleworks.com/fest
-
 This page provides a basic overview of correct YAML syntax, which is how Ansible
 playbooks (our configuration management language) are expressed.  
 
